src/themes/default/elements/icon.overrides

```diff
@@ -561,7 +575,7 @@ i.icon.facebook.square:before { content: "\f082"; }
 i.icon.linkedin.square:before { content: "\f08c"; }
 i.icon.github.square:before { content: "\f092"; }
 i.icon.twitter:before { content: "\f099"; }
-i.icon.facebook:before { content: "\f09a"; }
+i.icon.facebook.f:before { content: "\f09a"; }
```

Comment:
I found a duplicated class name here. The new (proposed) icon name should be `facebook f`, being just the «f», and `facebook` being the official logo.